Metadata.yaml

```diff
@@ -0,0 +1,246 @@
+
+PublicationDate: 2020-12-07
+
+Doi: 10.5281/zenodo.4290405
+
+ZenodoId: 4290405
+
+Description: |
+  SpECTRE is an open-source code for multi-scale, multi-physics problems
+  in astrophysics and gravitational physics. In the future, we hope that
+  it can be applied to problems across discipline boundaries in fluid
+  dynamics, geoscience, plasma physics, nuclear physics, and
+  engineering. It runs at petascale and is designed for future exascale
+  computers.
+
+  SpECTRE is being developed in support of our collaborative Simulating
+  eXtreme Spacetimes (SXS) research program into the multi-messenger
+  astrophysics of neutron star mergers, core-collapse supernovae, and
+  gamma-ray bursts.
+
+Keywords:
+  - Software
+  - Astrophysics
+  - General Relativity
+  - Numerical Relativity
+  - Multiphysics
+  - Gravitational Waves
+  - Discontinuous Galerkin
+  - Finite Difference
+  - Finite Volume
+
+Affiliations:
+  - &aei
+    Max Planck Institute for Gravitational Physics (Albert Einstein Institute),
+    Am Mühlenberg 1, Potsdam 14476, Germany
+  - &caltech
+    Theoretical Astrophysics, Walter Burke Institute for Theoretical
+    Physics, California Institute of Technology, Pasadena, CA 91125, USA
+  - &cornell
+    Cornell Center for Astrophysics and Planetary Science, Cornell University,
+    Ithaca, New York 14853, USA
+  - &fullerton
+    Nicholas and Lee Begovich Center for Gravitational-Wave Physics and
+    Astronomy, California State University, Fullerton, Fullerton, California
+    92831, USA
+  - &new_hampshire
+    Department of Physics & Astronomy, University of New Hampshire, 9 Library
+    Way, Durham NH 03824, USA
+
+Authors:
+  Description: |
+    We maintain three tiers of authors for the code as a whole: Core, Developers
+    and Contributors. The full author list for the SpECTRE DOI is built by
+    concatenating all authors in the order they are listed here.
+
+    The primary goal of this author list is to recognize infrastructure work
+    that does not or cannot have any science papers and results associated with
+    it. Through authorship on the SpECTRE DOI such work gains recognition that
+    holds value in academia. Authorship on the SpECTRE DOI does not directly
+    entitle a person for authorship rights on other publications. Authorship
+    rights on publications related to SpECTRE are governed by our publication
+    policy.
+
+    Note that the placement of authors in these tiers is inherently a subjective
+    topic since we are forced to compare different projects people work on.
+    Nevertheless, having guidelines and some way of promoting healthy and
+    constructive discussions is vital for ensuring the collaborative aspect of
+    the code is healthy.
+
+  Core:
+    Description: |
+      For the time a Developer serves as Core developer they take on some
+      project-management responsibilities, such as merging pull-requests.
+
+      Guidelines on becoming a Core developer:
```

Comment:
This list reads very awkwardly to me.  I suggest this header be changed to "Guidelines on Core developers:"...

Reply:
I reworded the list slightly, though didn't change the header because I think it would be good to keep it consistent with the other headers. Happy to adjust further if you think it's necessary.

Reply:
I don't like the header because the fourth bullet is not a guideline on becoming a core developer.

This whole list still reads very awkwardly.  (The other sections also somewhat, but less so than here.)  I think it is the mixture of imperative and declarative bullets.

Reply:
Ok, restructured the description a bit. How do you like it now?

Reply:
This reads much better, thanks.  I think we should keep the "Most people will not end up in the Core developer..." part somewhere if we're still planning for developer tier to have authorship implications, though.

Reply:
I thought the paragraph above makes the authorship implications sufficiently clear:

> We maintain three tiers of authors for the code as a whole: Core, Developers and Contributors. The full author list for the SpECTRE DOI is built by concatenating all authors in the order they are listed here.
>
> The primary goal of this author list is to recognize infrastructure work that does not or cannot have any science papers and results associated with it. Through authorship on the SpECTRE DOI such work gains recognition that holds value in academia. Authorship on the SpECTRE DOI does not directly entitle a person for authorship rights on other publications. Authorship rights on publications related to SpECTRE are governed by our publication policy.

Once we add our publication policy to the repo we can add a link here. Writing the publication policy is a bit out of scope for this PR, and we should do that after the SXS policies are finished.

Don't you think this is enough explanation for now?

Reply:
I think that's enough about the policy, but as I said, if developer tier has authorship implications (which I now see that it still does), then I think we should retain the part about most people not becoming core developers.  It's different from the other tiers, where moving up to the tier is expected to happen eventually for anyone continuously working on SpECTRE.

Reply:
I agree with @wthrowe, especially because as time goes on fewer and fewer people will move to Core Devs simply because there will be a lot less infrastructure work to do.

Reply:
@nilsdeppe @wthrowe added back the paragraph, please check the fixup